identity-service/src/config.js

```diff
@@ -235,12 +241,24 @@ const config = convict({
     env: 'minimumBalance',
     default: null
   },
+  minimumRelayerBalance: {
+    doc: 'Minimum token balance for relayer below which /balance_check fails',
+    format: Number,
+    env: 'minimumRelayerBalance',
+    default: null
+  },
   ethMinimumBalance: {
     doc: 'Minimum ETH balance below which /eth_balance_check fails',
     format: Number,
     env: 'ethMinimumBalance',
     default: 0.5
   },
+  minimumFunderBalance: {
+    doc: 'Minimum eth balance for funder below which /eth_balance_check fails',
+    format: Number,
```

Comment:
eth may need to be fractional. since eth is so expensive we may want to do like 0.5?

Reply:
Yes, but we do a parse float for this number, so this should already be supported.

I've added a default of 0.5 here.